lib/jnpr/junos/utils/start_shell.py

```diff
@@ -100,18 +112,20 @@ def open(self):
         """
         if self.ON_JUNOS is True:
             self._chan = subprocess.Popen(
-                ["cli", "start", "shell"],
+                ["cli", "start", "shell", self.shell_type],
                 shell=False,
                 stdin=subprocess.PIPE,
                 stdout=subprocess.PIPE,
+                close_fds=1,
+                bufsize=1,
             )
         else:
             self._client = open_ssh_client(dev=self._nc)
             self._chan = self._client.invoke_shell()
 
             got = self.wait_for(r"(%|>|#|\$)")
             if got[-1].endswith(_JUNOS_PROMPT):
-                self.send("start shell")
+                self.send("start shell self.shell_type")
```

Comment:
You need to concatenate two string

Reply:
Thanks for the review comments , 
Fixed the command 
self.send("start shell " + self.shell_type)